spec/models/manageiq/providers/kubernetes/container_manager/refresher_spec.rb

```diff
@@ -274,6 +277,36 @@ def assert_specific_container_project
     @container_pr.ext_management_system.should == @ems
   end
 
+  def assert_specific_container_quota
+    container_quota = ContainerQuota.find_by_name("quota")
+    container_quota.creation_timestamp.kind_of?(ActiveSupport::TimeWithZone)
+    container_quota.container_quota_items.count.should == 8
+    cpu_quota = container_quota.container_quota_items.select { |x| x[:resource] == 'cpu' }[0]
+    cpu_quota.should have_attributes(
+      :quota_desired  => '20',
+      :quota_enforced => '20',
+      :quota_observed => '100m',
+    )
+  end
+
+  def assert_specific_container_limit
```

Comment:
From previous bugs - It's always a good idea to assert the relation (e.g. - the new limit/quota actually belongs to the right project)